### PR TITLE
Add a confirmation popup on Thread delete action

### LINKED ISSUE #321: feature request: put a timed undo delete button to prevent people from deleting all messages of a one contact by accident
Like in the "Dekko"-app I suggest putting in a timed undo button in the messaging-app of ubuntu touch. Some silly people like me sometimes swipe by accidentone of their contacts to the right and then press the delete-button. If there were an undo button ore a question like: do you really want to delete the messaging history of this contackt? this might be very helpful.

### PR DESCRIPTION
**Pull request by lduboeuf**
fixes #321 

in replacement of #322 
[EDIT]
Capture d’écran de 2022-03-29 09-40-47

### CODE REVIEW

**Comment by cibersheep** [comment]
Does this Popup miss the Title?

**Comment by lduboeuf** [comment]
> Does this Popup miss the Title?

Yes, i never know what in this case to put on the title

**Comment by cibersheep** [comment]
> > Does this Popup miss the Title?
> 
> Yes, i never know what in this case to put on the title

The general action? Some like: «Delete Message» / «Delete Messages»?

**Comment by lduboeuf** [comment]
@cibersheep should be better now . I admit, without a title, the popup was not looking great ;-)

**Comment by cibersheep** [comment]
> @cibersheep should be better now . I admit, without a title, the popup was not looking great ;-)

Hi hi hi... awesome :*

File: po/messaging-app.pot [hunk 1]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-01-17 08:59+0000\n"
+"POT-Creation-Date: 2022-03-29 07:40+0000\n"
```
Comment (Flohack74) [inline po/messaging-app.pot]: Seems pot file does not contain the new strings?

**Review by Flohack74 — CHANGES_REQUESTED** [review]

File: po/messaging-app.pot (hunk 1 above)
Comment (lduboeuf) [reply po/messaging-app.pot]: true, sorry, done.

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**